### DISCUSSION TITLE
board of notes / tickets

### DISCUSSION THREAD

**Discussion by b4zz4** [tin2tin/Blender_Screenwriter#45] [Ideas]
![](https://download.tuxfamily.org/4232/wordpress/2021/02/4232_tablero_corcho.jpg)

Acts, headers and description of action scene.
It would be great if you exchange the notes, you're reordering the script.

I think this would be similar to the [MindMapper](https://github.com/SpectralVectors/MindMapper) or [split-frame-from-text](https://github.com/gabrielmontagne/blender-addon-split-frame-from-text) program:

![2021-07-03_10-25_45](https://user-images.githubusercontent.com/643833/124355966-22f14700-dbea-11eb-9d4b-643d6876a9dd.jpg)

**Comment by tin2tin** [comment]
Are you familiar with this: https://github.com/gabrielmontagne/blender-addon-marker-preview-video/issues/1#issuecomment-770417986
Or this: https://blenderartists.org/t/blender-screenwriter/1190208/37

Some of the main problems are how to make the order change the order in the text editor? 
How to update then text nodes?
Should the nodes only display the synopsis?
How to deal with the inability to work in multiple lines?
How to get changes back into the text editor
How to detect the order of the strips?
Are anyone actually going to do mind mapping for screenplays in Blender?
(and more)

**Comment by b4zz4** [comment]
I only thought for each act a group and for each scene a node and the beginning of the action text. This only serves to visualize the film at a simple glance. Not to edit it.


But if it's a problem to check the order of the nodes, so I thought they were linked or in a cricket, it's the hard thing...

**Comment by b4zz4** [comment]
Interesting project and partnering with blender also

https://notabug.org/jyamihud/VCStudio

**Comment by tin2tin** [comment]
I didn't know that it existed, but they're not supporting Windows....

For a powerful and free screenwriting tool, check this out: https://www.scrite.io/

**Comment by b4zz4** [comment]
Just take it as a reference, I like the idea of an addons and compatible with fountain


El martes, 20 de julio de 2021 19:56:45 ART, tin2tin ***@***.***> escribió: 







I didn't know that it existed, but they're now supporting Windows....

For a powerful and free screenwriting tool, check this out: https://www.scrite.io/
